src/main/resources/graphql/schema.graphqls

```diff
@@ -354,6 +354,9 @@ type Feature {
 type MultiPolygon {
     type: String
     coordinates: [[[[Float]]]]
+
+    "coordinates encoded with mapbox polyline, precision 5"
+    encodedCoordinates: [[String]]
```

Comment:
I would prefer this to be in properties instead, and as a custom type with a custom scalar for Polyline instead of String.

Reply:
Moved to properties now, some typing remaining